__tests__/integration-metrics.test.js

```diff
@@ -167,9 +195,9 @@ describe('Metrics SDK Integration Tests', () => {
     expect(response.content.text.replace('\n', '')).toBe(JSON.stringify({ message: 'hello world' }));
     // The \n character above means we cannot compare to a fixed number
     expect(response.content.size).toStrictEqual(response.content.text.length);
-    expect(response.content.mimeType).toBe('application/json; charset=utf-8');
+    expect(response.content.mimeType).toMatch(/application\/json(;\s?charset=utf-8)?/);
```

Comment:
Doing `response()->json(['message' => 'hello world'])` in Laravel doesn't add a charset to the header. It'll get that if you do the following instead but for the purpose of my demo app I didn't think it was neceesary.

```php
return response()->json(['message' => 'hello world'], 200, [
  'Content-Type' => 'application/json;charset=UTF-8', 
  'Charset' => 'utf-8'
], JSON_UNESCAPED_UNICODE);
```

// https://stackoverflow.com/questions/50717005/laravel-encode-json-responses-in-utf-8